src/data/events.json

```diff
@@ -197,5 +197,16 @@
     "location": "Chennai",
     "communityName": "Kotlin User Group Chennai",
     "communityLogo": "https://i.ibb.co/N6q8WqzK/KUG-Board-Logo-1.png"
+  },
+    {
+    "eventName": "Chennai Web Engineering - December",
+    "eventDescription": "Unique opportunity to exchange perspectives, gain industry insights, and network with other tech-minded individuals passionate about the cutting edge of Artificial Intelligence.",
+    "eventDate": "2025-12-13",
+    "eventTime": "10:30",
+    "eventVenue": "HTC Global Services, Guindy",
+    "eventLink": "https://luma.com/lma4uoc7",
+    "location": "Chennai",
+    "communityName": "Chennai Web Engineering",
+    "communityLogo": "https://l1nq.com/T7fu2"
```

Comment:
Hey @chennaiwebengineering 
please change the communityLogo to imgbb since the current one is not an available hostname in our next config.
Thanks. Happy to merge it once thats done
Cheers :)